### DISCUSSION TITLE
Global `expect` vs `expect` in test context

### DISCUSSION THREAD

**Discussion by joelmukuthu** [vitest-dev/vitest#4362] [Q&A] [answered]
Is there a general recommendation of which `expect` to use when writing tests? To be clear, I mean this:

```
import { expect as globalExpect, test } from 'vitest';

test('global `expect`', () => {
  globalExpect(1).toBe(1);
});

test('test-context `expect`', ({ expect: testContextExpect }) => {
  testContextExpect(1).toBe(1);
});
```

The [docs](https://vitest.dev/guide/test-context.html#built-in-test-context) point out that the `expect` from the test context is

> useful for running snapshot tests concurrently because global expect cannot track them. 

I read that as a +1 for the test-context `expect`. Are there downsides that I should be aware of if I'm writing tests with the test-context `expect`?

**Comment by sheremet-va** [comment, marked as answer]
Creating a new expect for every test is a bit of an overhead. If you don't use a local expect, we don't create a new one.

**Reply by joelmukuthu** [reply]
Great to know, thanks!
